checker/manual/creating-a-checker.tex

```diff
@@ -194,27 +193,31 @@ \subsection{Defining the type qualifiers\label{define-type-qualifiers}}
 
 \begin{Verbatim}
   // Define an annotation for the @NonNull type qualifier.
-  @TypeQualifier
   @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
   public @interface NonNull { }
 \end{Verbatim}
 
 \noindent
-Write the \refqualclass{framework/qual}{TypeQualifier} meta-annotation on the annotation definition
-to indicate that the annotation represents a type qualifier
-and should be processed by the checker.
-Also write a \sunjavadocanno{java/lang/annotation/Target.html}{Target}
-meta-annotation to indicate where the annotation may be written.
-(An annotation that is written on an annotation
-definition, such as \refqualclass{framework/qual}{TypeQualifier}, is called a \emph{meta-annotation}.)
+
+Write a \sunjavadocanno{java/lang/annotation/Target.html}{Target}
+meta-annotation on the annotation definition to indicate where the annotation
+may be written. All type qualifiers that users can write in source code should
+have the value \<ElementType.TYPE\_USE> (and optionally with the additional value of
+\<ElementType.TYPE\_PARAMETER>, but no other \<ElementType> values).
+The Checker Framework automatically treats any annotation that has this value
```

Comment:
A checker only treats annotations with \@Target(TYPE_USE) as a type qualifier if it is declared in the qual package in that same package as the checker, right?